lib/pharos/scripts/kubeadm-reconfigure.sh

```diff
@@ -4,5 +4,6 @@ set -eu
 
 unset http_proxy HTTP_PROXY HTTPS_PROXY
 
+kubeadm alpha phase certs apiserver --config ${CONFIG}
```

Comment:
It just logs  `Using the existing %s certificate and key` if the cert exists and is signed by the CA:

* https://github.com/kubernetes/kubernetes/blob/v1.11.1/cmd/kubeadm/app/cmd/phases/certs.go#L186-L191
* https://github.com/kubernetes/kubernetes/blob/v1.11.1/cmd/kubeadm/app/phases/certs/certs.go#L102
* https://github.com/kubernetes/kubernetes/blob/v1.11.1/cmd/kubeadm/app/phases/certs/certs.go#L561

Interestingly, the implementation is commented as behaving differently:

https://github.com/kubernetes/kubernetes/blob/v1.11.1/cmd/kubeadm/app/phases/certs/certs.go#L88

> ```
> // If the apiserver certificate and key files already exists in the target folder, they are used only if evaluated equal; otherwise an error is returned.
> ```

https://github.com/kubernetes/kubernetes/blob/b1b29978270dc22fecc592ac55d903350454310a/cmd/kubeadm/app/phases/certs/certs.go#L539-L541

> ```
> // If there already is a certificate file at the given path; kubeadm tries to load it and check if the values in the
> // existing and the expected certificate equals. If they do; kubeadm will just skip writing the file as it's up-to-date,
> // otherwise this function returns an error.
> ```

However, I don't spot anything in the code that would actually do that, it doesn't match the observed behavior either...